src/ValueObjects/TransformationV2.php

```diff
@@ -115,6 +115,8 @@ private function renameInputMappingKeys(array $inputMappingTable, ?string $keyPr
                 $switchKey = $k;
             }
             switch ($switchKey) {
+                case 'loadType': // skip this config
+                    continue 2;
```

Comment:
proc skip config?  nemuzem jen vyhodit ten klic?

Reply:
promin, uz vidim ze ignoruje jen ten parametr a ne celej config :-)

Reply:
skipne se tohle nastavení, takže do nových transfromací se nedostane

Reply:
ok, I'm going to approve it :-)